_component_design/dropdowns.md

```diff
@@ -15,20 +15,20 @@ need: selectors
 
 {% include todo.html content="Update style and add additional examples" %}
 
-Use the simple selector when choosing from a limited number options. If more
-than 10-12 options need to displayed, use the complex selector, which contains
-a search. Always use the complex selector for devices. Be mindful of the length
+Use the simple selector when choosing from a limited number of options. Use the
+complex selector if more than 10-12 options need to displayed.
```

Comment:
Is a 'simple selector' or 'complex selector' a design thing, @technabors ?

Reply:
It is not commonly used design language and looks like a delineation created by the original content creator. I am not attached the nomenclature, although it might make sense if they had provided an example, which they didn't. IMO we omit this reference until there is an example at minimum or just remove altogether since its confusing

Reply:
Okay - what do you recommend for the change? Because we need to define between the two types of selectors.

Reply:
@robb-romans - this is why. After I commented on this, @technabors suggested that we remove this para for now, until we are able to clarify further later down the track.